unixfs/examples/cat.rs

```diff
@@ -0,0 +1,161 @@
+use cid::Cid;
+use ipfs_unixfs::file::{visit::IdleFileVisit, FileReadFailed};
+use std::convert::TryFrom;
+use std::fmt;
+use std::io::{Error as IoError, Read, Write};
+use std::path::PathBuf;
+
+fn main() {
+    let cid = match std::env::args().skip(1).next().map(|s| Cid::try_from(s)) {
+        Some(Ok(cid)) => cid,
+        Some(Err(e)) => {
+            eprintln!("Invalid cid given as argument: {}", e);
+            std::process::exit(1);
+        }
+        None => {
+            eprintln!("USAGE: {} CID", std::env::args().next().unwrap());
+            eprintln!();
```

Comment:
nit: add an `\n` at the end of the first `eprintln!`

Reply:
I had imagined that this uses CRLF on windows but apparently not [println!](https://doc.rust-lang.org/std/macro.println.html).